specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iotdps.json

```diff
@@ -0,0 +1,2622 @@
+            "type": "string",
+            "default": "true"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The status of the long running operation.",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationResult"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/skus": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Get the list of valid SKUs for a provisioning service.",
+        "description": "Gets the list of valid SKUs and tiers for a provisioning service.",
+        "operationId": "IotDpsResource_listValidSkus",
+        "x-ms-examples": {
+          "DPSGetValidSku": {
+            "$ref": "./examples/DPSGetValidSku.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "provisioningServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of provisioning service."
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of resource group."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is a synchronous operation. The body contains a JSON-serialized array of the valid SKUs for this provisioning service.",
+            "schema": {
+              "$ref": "#/definitions/IotDpsSkuDefinitionListResult"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/certificates/{certificateName}/generateVerificationCode": {
+      "post": {
+        "tags": [
+          "POST"
+        ],
+        "operationId": "DpsCertificate_GenerateVerificationCode",
+        "description": "Generate verification code for Proof of Possession.",
+        "x-ms-examples": {
+          "DPSGenerateVerificationCode": {
+            "$ref": "./examples/DPSGenerateVerificationCode.json"
```

Comment:
Adding more examples with the supported query strings will be helpful for customers.

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)